goosebit/db.py

```diff
@@ -22,6 +23,10 @@ async def init():
     await command.migrate()
     await command.upgrade(run_in_transaction=True)
     await Tortoise.generate_schemas(safe=True)
+    for update in await FirmwareUpdate.all():
```

Comment:
Instead of `all()` better to filter on db using `filter()`.

Reply:
Not sure what to filter by here, I suppose maybe `uri` starts with `file`?

Reply:
Thought about using `await Firmware.filter(url__startswith='file').all()`. Debatable if a good idea, as logic of `local` is the implemented in two different places. On the upside: with a lot of remote firmware it will be very fast.